components/atoms/ModalHeader.jsx

```diff
@@ -17,7 +19,8 @@ ModalHeader.propTypes = {
     title: PropTypes.string,
     onClose: PropTypes.func,
     className: PropTypes.string,
-    style: PropTypes.object
+    style: PropTypes.object,
+    size: PropTypes.string
```

Comment:
```
size: PropTypes.oneOf( 'sm']),
```
so it's clear that's the only accepted size

Reply:
ahhh cooo.